management-gateway/src/main/java/org/frankframework/management/gateway/events/ClusterMemberEvent.java

```diff
@@ -0,0 +1,43 @@
+/*
+   Copyright 2024 WeAreFrank!
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+*/
+package org.frankframework.management.gateway.events;
+
+import org.frankframework.management.bus.OutboundGateway.ClusterMember;
+import org.springframework.context.ApplicationContext;
+
+import jakarta.annotation.Nonnull;
+import lombok.Getter;
+
+public class ClusterMemberEvent extends AbstractGatewayEvent {
+
+	public enum Type {
```

Comment:
Type is very generic. Does `ActionType` (or something better) fits the purpose even more?